recipes/default.rb

```diff
@@ -17,6 +17,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# This resource will download the Handle.exe tool required from the specified
+# URL. If in an airgapped environment, you will need to either override the
+# ['chef_client_updater']['handle_download_url'] attribute or alternatively
+# ensure the executable files are extracted to Chef::Config[:file_cache_path]
+remote_file "#{Chef::Config[:file_cache_path]}/handle.zip" do
```

Comment:
This logic should go into the updater resource so that users can use it without the need for the recipe. The attribute should also be a property on the resource instead of a node attribute.